components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dto/OAuthAppRevocationRequestDTO.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.dto;
+
+/**
+ * OAuth application token revocation request dto.
+ */
+public class OAuthAppRevocationRequestDTO {
+    private String applicationName;
```

Comment:
Formatting

Reply:
Addressed in [40fde89](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/40fde89cb099493c8f2a302b34b1141d9aea3e8d)